src/pkg/projects/depproj.targets

```diff
@@ -0,0 +1,109 @@
+      </FilesToPackage>
+    </ItemGroup>
+
+    <!-- on windows workaround max-path -->
+    <!-- this can be removed once we build on core MSBuild which supports long paths. -->
+    <ItemGroup Condition="'$(OsEnvironment)'=='Windows_NT'">
+      <!-- choose a shorter path name, excluding package version -->
+      <_shortSourcePath Include="@(_sourcePath->'$(ObjDir.Replace('/', '\'))src-%(NuGetPackageId)')">
+        <Original>%(_sourcePath.FullPath)</Original>
+      </_shortSourcePath>
+      <_sourcePath Remove="@(_sourcePath)" />
+      <_sourcePath Include="@(_shortSourcePath)" />
+    </ItemGroup>
+    <RemoveDir Condition="'@(_shortSourcePath)' != '' AND Exists('%(_shortSourcePath.Identity)')" Directories="%(_shortSourcePath.Identity)" />
+    <Exec Condition="'@(_shortSourcePath)' != ''" Command="mklink /J %(_shortSourcePath.Identity) %(_shortSourcePath.Original)" />
+
+    <ItemGroup>
+      <FilesToPackage Include="%(_sourcePath.Identity)\%2A%2A\%2A.%2A">
+        <TargetPath>src</TargetPath>
+        <IsSourceCodeFile>true</IsSourceCodeFile>
+      </FilesToPackage>
+    </ItemGroup>
+  </Target>
+
+  <Target Name="GetFilesToPackage"
+          DependsOnTargets="ResolveNuGetPackages;GetFilesFromPackages"
+          Returns="@(FilesToPackage)" />
+
+  <Target Name="GetDependenciesToPackage"
+          Condition="'@(DependenciesToPackage)' != ''"
+          DependsOnTargets="ResolveNuGetPackages"
+          Returns="@(_DependenciesToPackageWithVersion)">
+    <ItemGroup>
+      <!-- intersect ReferencedPackage with DependenciesToPackage -->
+      <_DependenciesToPackageWithVersion Include="@(PackageReference)" Condition="'@(PackageReference)' == '@(DependenciesToPackage)' AND '%(Identity)' != ''">
+        <TargetFramework>$(PackageTargetFramework)</TargetFramework>
+      </_DependenciesToPackageWithVersion>
+    </ItemGroup>
+
+    <Message Importance="low" Text="%(_DependenciesToPackageWithVersion.Identity) : %(_DependenciesToPackageWithVersion.Version) : %(_DependenciesToPackageWithVersion.TargetFramework)" />
+  </Target>
+
+  <!-- Creates the *.versions.txt file describing where data in this package came from. -->
+  <Target Name="GenerateHashVersionsFile" DependsOnTargets="GetDependencyVersionFiles">
+    <ItemGroup>
+      <DependencyVersionFile Content="$([System.IO.File]::ReadAllText('%(Identity)').Trim())" />
+
+      <_VersionsFileLines Include="core-setup $(LatestCommit)" />
+      <_VersionsFileLines Include="@(DependencyVersionFile ->'%(Name) %(Content)')" />
+    </ItemGroup>
+
+    <WriteLinesToFile Lines="@(_VersionsFileLines)"
+                       File="$(IntermediateOutputPath)\$(FrameworkPackageName).versions.txt"
+                       Overwrite="true"/>
+  </Target>
+
+  <!-- Target overrides (can't be shared with pkgproj) -->
+
+  <Target Name="Build" DependsOnTargets="GenerateHashVersionsFile;GetFilesToPackage" />
+
+  <!-- remove when fixing https://github.com/dotnet/buildtools/issues/1273 -->
```

Comment:
Still applicable?

Reply:
It looks like it actually isn't an override, but it's still used. Removing it makes the pkg build fail during `GenerateRuntimeDependencies` because `DetermineRuntimeDependencies` isn't defined in the depproj. (Since only pkgproj gets buildtools `Packaging.targets`.)

I don't follow (1) and (2) in the buildtools issue, it looks like (3) hasn't happened, though: "We call DetermineRuntimeDependencies on all references in the lineup package but should only call it on pkgproj references."

Reply:
I see, bummer.  Sorry for that.